actsims/signal.py

```diff
@@ -9,16 +9,13 @@
 
 class SignalGen(object):
     # a helper class to quickly generate sims for given patch
-    def __init__(self, cmb_type='LensedCMB', dobeam=True, add_foregrounds=True, apply_window=True, max_cached=1, model="act_mr3", extract_region=None,extract_region_shape=None, extract_region_wcs=None, apply_rotation=False, alpha_map=None, add_poisson_srcs = False):
+    def __init__(self, cmb_type='LensedCMB', dobeam=True, add_foregrounds=True, apply_window=True, max_cached=1, model="act_mr3", apply_rotation=False, alpha_map=None, add_poisson_srcs = False):
```

Comment:
add_poisson_srcs does nothing in initialization, so I think it can instead be an argument in e.g. get_signal_sim rather than a member boolean of the class.

Reply:
For Gaussian foregrounds, we do set a global flag "add_foregrounds" in initialization, but we allow to override when you call "get_signal_sim". Maybe, we do something similar for pt srcs?

Reply:
`add_poisson_srcs `is now get_signal_sim argument